### ISSUE TITLE
Expl3 control sequences in curly braces are not highlighted properly

### ISSUE THREAD

**Issue by yudai-nkt** [yudai-nkt/language-tex#9]
In 4e6e6eb4e27ca539c687cc8b0d9d14d3650f72da, I followed the naming scheme in LaTeX3 programming documented [here](http://mirrors.ctan.org/macros/latex/contrib/l3kernel/expl3.pdf) and support for `expl3` package has been much improved:

``` latex
\ExplSyntaxOn
\module_foo:nnn
\__module_foo:nnn
\l_module_foo_tl
\l__module_foo_tl
\ExplSyntaxOff
```

will look like this (variables might look pale, but they are indeed highlighted).

![2016-08-21 18 50 37](https://cloud.githubusercontent.com/assets/13291527/17836508/432e8084-67d0-11e6-90aa-89755b81dfeb.png)

When control sequences are inside braces, however, syntax highlighting breaks and LaTeX2e rules are applied.

``` latex
\ExplSyntaxOn
{\module_foo:nnn}
{\__module_foo:nnn}
{\l_module_foo_tl}
{\l__module_foo_tl}
\ExplSyntaxOff
```

![2016-08-21 18 55 06](https://cloud.githubusercontent.com/assets/13291527/17836543/0aad52ac-67d1-11e6-85af-03c9cb67a4e0.png)

**Comment by yudai-nkt** [comment]
Deleting [`meta.group.braces.tex` attribute](https://github.com/yudai-nkt/language-tex/blob/4e6e6eb4e27ca539c687cc8b0d9d14d3650f72da/grammars/tex.cson#L67-L82) addresses this problem.

![2016-08-22 9 03 54](https://cloud.githubusercontent.com/assets/13291527/17840644/79687926-6847-11e6-8d8e-84d841675610.png)

I suppose this doesn't have side effects, but am not 100% sure.

**Comment by yudai-nkt** [comment]
Deleting [`meta.group.braces.tex` attribute](https://github.com/yudai-nkt/language-tex/blob/4e6e6eb4e27ca539c687cc8b0d9d14d3650f72da/grammars/tex.cson#L67-L82) addresses this problem.

![2016-08-22 9 03 54](https://cloud.githubusercontent.com/assets/13291527/17840644/79687926-6847-11e6-8d8e-84d841675610.png)

I suppose this doesn't have bad side effects, but am not 100% sure.

**Comment by yudai-nkt** [comment]
Reopen this issue as the closing commit was reverted to fix #11.

**Comment by yudai-nkt** [comment]
Closing this because I'm now a collaborator of [`language-latex`](https://github.com/area/language-latex/) package.
